Dockerfile

```diff
@@ -1,14 +1,12 @@
-FROM elifesciences/php_cli
+ARG image_tag=latest
+FROM elifesciences/proofreader-php_composer:${image_tag} AS build
+FROM php:7.0.29-cli-alpine
 
-USER elife
-RUN mkdir -p /srv/proofreader-php
-WORKDIR /srv/proofreader-php
-COPY --chown=elife:elife composer.json /srv/proofreader-php/
-RUN composer install --classmap-authoritative --no-dev
-COPY --chown=elife:elife . /srv/proofreader-php
+RUN apk add --no-cache bash
 
-USER root
-RUN touch .php_cs.cache && chown www-data:www-data .php_cs.cache
+ENV PROJECT_FOLDER=/srv/proofreader-php
+RUN mkdir ${PROJECT_FOLDER}
+WORKDIR ${PROJECT_FOLDER}
 
-USER www-data
-CMD bin/proofreader
+COPY bin/ bin/
```

Comment:
I think this will be owned by `root`, which would still make it read only for the `www-data` user

Reply:
In general the use of builder images should make the `elife` user less important as they have their own user for running processes like `composer`